backend/src/shared/api/types/PlayerSit.ts

```diff
@@ -3,7 +3,7 @@ export type PlayerSitPayload = {
   socketId: string;
 };
 
-export type PlayerSitResult = {
-  seatNumber: string;
-  socketId: string;
+export type PlayerSitOutput = {
+  ok: boolean,
+  error: string
```

Comment:
```suggestion
  error?: string
```
Using the optional property `error?` we only have to send an error when we have one. For example you could omit the empty error [here](https://github.com/richardaspinall/poker-react-node/blob/2f6d9b906d7802d9fd508217eaf82189c2e0df3a/backend/src/routes/actions.ts#L51) with the above change: